roxie/ccd/ccdmain.cpp

```diff
@@ -1035,6 +1037,9 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
         mergeSlaveStatistics = topology->getPropBool("@mergeSlaveStatistics", true);
         defaultCollectFactoryStatistics = topology->getPropBool("@collectFactoryStatistics", true);
         defaultNoSeekBuildIndex = topology->getPropBool("@noSeekBuildIndex", isContainerized());
+        parallelLoadQueries = topology->getPropInt("@parallelLoadQueries", 8);
+        if (!parallelLoadQueries)
+            parallelLoadQueries = 1;
 
```

Comment:
Given mark's comment about 2 cores, probably worth limiting the maximum to getAffinityCpus().

Reply:
Maybe, though as queries quite often block making calls to dali, it may be useful to overcommit too..

Reply:
His comments suggested that increasing the value from 4 to 8 improved the speed, even with only 2 CPUs...

Reply:
ok